decks/contributing-to-ansible.html

```diff
@@ -0,0 +1,434 @@
+                  <li>Does it follow the CRUD principles (Create, Read, Update, Delete)?</li>
+              </ul>
+          </ul>
+          <aside class="notes">
+              <ul>
+                  <li>One thing I think is often overlooked here is that you don't need to be a programer, or have any understanding of the internals to do a functional or documentation review.</li>
+                  <li>We know the "how to review" parts of Ansible's Docs need some work, what would you like to see there? - give comments in Chat or join in #ansible-community</li>
+                  <li>I accept some of this is subjective, so we are looking for a net improvement<li>
+              </ul>
+          </aside>
+        </section>
+
+        <section class="text-center">
+          <h2>Development Process: How to review - example</h2>
+          <ul>
+              <li>FIXME: Screenshot/code-snippit of a good feedback</li>
+              <li>FIXME: Details of what and how was tested, mindful comments</li>
+              <li></li>
+              <li></li>
+              <li></li>
+          </ul>
+          <aside class="notes">
+              <ul>
+                  <li>So lets see this in action</li>
+              </ul>
+          </aside>
+        </section>
+
+        <section class="text-center">
+          <h2>Development Process: Finding PRs</h2>
+          <ul>
+              <li>GitHub Search</li>
+              <ul>
+                  <li></li>
+                  <li></li>
+              </ul>
+              <li>GitHub Labels</li>
+              <ul>
+                  <li>github.com/ansible/ansible/labels to find labels</li>
+                  <li>directory name are often labels lib/modules/cloud/docker/docker_image = module, cloud, docker</li>
+                  <li>bug, feature, module, new_module, docs, test</li>
+              </ul>
+              <li>https://ansible.sivel.net/pr/byfile.html</li>
+              <ul>
+                  <li>Search by filename (or directory) to find PRs</li>
+                  <li>Updated every few hours</li>
+              </ul>
+          </ul>
+          <aside class="notes">
+              <ul>
+                  <li>As we talked about earlier, review Issues and PRs, with whatever skills you bring <b>is</b> really useful, so where do <b>you</b> start</li>
+              </ul>
+          </aside>
+        </section>
+<!-- END: PR Review -->
+
+        <section class="text-center">
+          <h2>Working Groups</h2>
+          <ul>
+              <li>Group of people with a common interest, improving Ansible's support for X</li>
+              <li>Generally existing users of X</li>
+              <li>Trying to solve a problem: Fix bugs, add new functionality</li>
+              <li>Maybe spread across the globe</li>
+              <li>Communicate via GitHub Issues/PRs/Wiki</li>
+              <li>May have a dedicated IRC channel</li>
+              <li>May have IRC Meetings (recorded)</li>
```

Comment:
might be worth mentioning somewhere, probably in your notes how people can get the recording if they want it. also make a note of where the times are discussed.

Reply:
> might be worth mentioning somewhere, probably in your notes how people can get the recording if they want it

Good idea, done


> where the times are discussed

Not sure what is meant by this.

Reply:
oops sorry @gundalow where the meeting times are discussed, in case people want to join them.